prometheus/prometheus.spec

```diff
@@ -12,7 +12,7 @@
 
 Name:           prometheus
 Version:        2.14.0
-Release:        3%{?dist}
+Release:        4%{?dist}
```

Comment:
Is there any actual change? If not, this package can stay at the same version as before.

Reply:
Shouldn't we increment it on every release?

Reply:
I don't see why we should.

Reply:
It could stay the same, however I rebuild all RPMs for all releases